include/photo_album.h

```diff
@@ -1,56 +1,64 @@
 #ifndef POKEHEARTGOLD_PHOTO_ALBUM_H
 #define POKEHEARTGOLD_PHOTO_ALBUM_H
 
+#include "camera.h"
 #include "save.h"
 
-#define PHOTO_ALBUM_MAX       36
+#define PHOTO_ALBUM_MAX 36
 
 typedef struct PHOTO_MON {
-    u16 species;
-    u8 form;
-    u8 shiny:1;
-    u8 gender:1;
```

Comment:
stick with 4 spaces for these, not 2 spaces

Reply:
???? where are you seeing 2 spaces here?

Reply:
disregard, original comment is outdated